subsys/net/lib/tftp/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
It seems that you've accidently removed TFTP library in https://github.com/zephyrproject-rtos/zephyr/pull/59669/commits/212bf4c5e931cc396ed0fa558ecce2b86b13607e, and then recommited it in https://github.com/zephyrproject-rtos/zephyr/pull/59669/commits/a2a34522d08a0ea825606f419100775feb7b3a90. We should not introduce such changes in the history, please amend the former commit so that it does not touch anything unrelated.